Zotero/Scenes/Detail/PDF/Views/PDFReaderViewController.swift

```diff
@@ -421,6 +421,14 @@ class PDFReaderViewController: UIViewController {
     // MARK: - Actions
 
     private func update(state: PDFReaderState) {
+        if state.changes.contains(.md5) {
+            DispatchQueue.main.async {
+                self.coordinatorDelegate?.showDocumentChangedAlert(key: state.key, parentKey: state.parentKey, libraryId: state.library.identifier)
+            }
+            close()
```

Comment:
It might be better if either `close()` has a completion block that it passes to `dismiss` and in that  `showDocumentChangedAlert()` is called,
or even if `showDocumentChangedAlert()` has an action block parameter, that then calls `close()`.
In the latter case, there is no need to have the `DetailCoordinator.navigationController` present, and it seems less forced than just dismissing the current screen.

Reply:
Changed in bab66baa9ff2f21e76412a20c678189eb7f53680